Apps/openvpn-as/docker-compose.yml

```diff
@@ -0,0 +1,161 @@
+# Configuration for OpenVPN-AS setup
+
+# Name of the OpenVPN-AS application
+name: big-bear-openvpn-as
+version: '1.0'
+
+# Service definitions for the OpenVPN application
+services:
+  # Service name: app
+  # The `app` service definition
+  app:
+
+     # Name of the container
+    container_name: OpenVPN
+
+     # Image to be used for the container
+    image: openvpn/openvpn-as:latest
```

Comment:
I would rather this be a version tag like: [2.13.1-d8cdeb9c-Ubuntu22](https://hub.docker.com/layers/openvpn/openvpn-as/2.13.1-d8cdeb9c-Ubuntu22/images/sha256-1525d59f85e92b17e3a0183e23133a27044acea27d9c2f792e2cc2fb8a92ac74?context=explore)

Reply:
I have just to change `latest` with it? Or is the one that I have to put on the json file?

Reply:
Change the latest to the versioned tag in config.json and the docker-compose. You shouldn't have to edit anything outside of config.json and docker-compose.yml anymore. I am trying to make it easy for contributions and maintenance because I was spending hours on everything.